src/index.tsx

```diff
@@ -88,7 +88,12 @@ export const ScreenStackHeaderSearchBarView = (
   props: React.PropsWithChildren<SearchBarProps>
 ): JSX.Element => <View {...props} />;
 
-export const ScreenStackHeaderConfig: React.ComponentType<ScreenStackHeaderConfigProps> = View;
+export const ScreenStackHeaderConfig: React.ComponentType<
```

Comment:
Because they were not previously exported from header config and I didn't want to change the interface. 

`ScreenStackHeaderConfigProps` is public and I these two methods are not exported.

Reply:
I changed it. I must have mistake them with some other prop